app/controllers/katello/api/v2/ostree_branches_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Katello
+  class Api::V2::OstreeBranchesController < Api::V2::ApiController
+    apipie_concern_subst(:a_resource => N_("an ostree branch"), :resource => "ostree_branch")
+    include Katello::Concerns::Api::V2::RepositoryContentController
+
+    private
+
+    def resource_class
+      OstreeBranch
+    end
+
+    def filter_by_content_view_filter(filter)
```

Comment:
This is a nonsensical addition though is it not? You can't filter by content view filters for ostree branches and someone looking at this code would think you could.